src/usr/local/www/system_advanced_network.php

```diff
@@ -84,6 +85,13 @@
 			unset($config['system']['prefer_ipv4']);
 		}
 
+		$_POST['global-v6duid'] = strtolower(str_replace("-", ":", $_POST['global-v6duid']));
```

Comment:
You should check if $_POST['global-v6duid'] is defined before try to access its value